modules/core/ante/ante_test.go

```diff
@@ -342,6 +343,20 @@ func (suite *AnteTestSuite) TestAnteDecorator() {
 			},
 			true,
 		},
+		{
+			"success on app callback error, app callbacks are skipped for performance",
+			func(suite *AnteTestSuite) []sdk.Msg {
+				suite.chainB.GetSimApp().IBCMockModule.IBCApp.OnRecvPacket = func(
+					ctx sdk.Context, packet channeltypes.Packet, relayer sdk.AccAddress,
+				) exported.Acknowledgement {
+					panic(fmt.Errorf("failed OnRecvPacket mock callback"))
```

Comment:
I checked running this test without the changes in the msg server results in test failure

Reply:
someone will likely open a PR to replace this with `errors.New()` since there's not formatting happening.. lol 

if you want to front run them go ahead, personally I don't care if its fmt or not here.